index.js

```diff
@@ -101,10 +101,13 @@ function Xray() {
           var $ = load(html, url);
           node.html($, next);
         });
-      } else {
+      } else if (source) {
```

Comment:
This `source` value is set to `null` in the `params` function when it does not pass either the `isURL` or `isHTML` test.